docs-ui/components/form.stories.js

```diff
@@ -401,7 +401,7 @@ storiesOf('Forms|Fields', module)
   )
   .add(
     'Without a parent Form',
-    withInfo('New form fields used withing having a parent Form')(() => {
+    withInfo('New form fields used without having a parent Form')(() => {
```

Comment:
Wow, that's impressive. What kind of automated typo fixing software are you using @minho42?

Reply:
Hi @joshuarli,
Thanks for reviewing.
I'm so new to git stuff, and haven't got time to figure out how to resolve the conflicts yet...

Anyway, I'm using my own script to find typos [https://github.com/minho42/typofinder](https://github.com/minho42/typofinder)